tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -6408,6 +6479,10 @@ void SpirvEmitter::storeValue(SpirvInstruction *lhsPtr,
   if (!lhsPtr || !rhsVal)
     return;
 
+  if (lhsPtr->getStorageClass() == spv::StorageClass::Input) {
+    emitError("cannot assign to input variable", loc);
+  }
+
```

Comment:
Is this still needed? Is the const qualification?

Reply:
It's not needed; I forgot to remove it.